### PR TITLE
feat: added filter & product list page

### PR DESCRIPTION
**Pull request by AtulHande03a**
filter contains: rating, price, category filter,sort by
added products list also.

### CODE REVIEW

**Review by ravikumar0403 — APPROVED** [review]
Looks good to me

File: css/products.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.filters {
+  position: sticky;
+  max-width: 18rem;
+  height: calc(100% - 18rem);
+  background-color: var(--color-primary-shade);
+
```
Comment (yashpatel1797) [inline css/products.css:6]: remove extra space

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Review by horsemaker — APPROVED** [review]
Suggestion: The products in the second row have a property of `space-around`, it's mostly not that & just `flex-start`.

**Comment by AtulHande03a** [comment]
> Suggestion: The products in the second row have a property of `space-around`, it's mostly not that & just `flex-start`.

made suggested changes

**Pull request merged**